packages/terra-responsive-element/tests/jest/__snapshots__/ResponsiveElement.test.jsx.snap

```diff
@@ -1,7 +1,3 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`should render a default component 1`] = `
-<div>
-  Default
-</div>
-`;
+exports[`should render a default component 1`] = `<div />`;
```

Comment:
I don't think any tests should have been affected by this change, was this file auto generated or manually updated?

Reply:
Auto Generated.

Reply:
This snapshot dropped the default text. I would try compiling and running the tests again. 

If this change is a result of the changes in ResponsiveElement.jsx it is non-passive and is not rendering the element. 

It looks like this test renders the initial div but never renders the defaultElement.

```
it('should render a default component', () => {
  const responsiveElement = shallow(<ResponsiveElement defaultElement={<div>Default</div>} />);
  expect(responsiveElement).toMatchSnapshot();
});
```

Reply:
It doesn't render the default text in the same frame, likely due to the call to requestAnimationFrame. Are we expecting the default text to be rendered at the same time as the initial div?

Reply:
The call to freeze the repaint breaks this test. My concern is that it will also cascade and break a significant amount of consumer tests down the chain.

Reply:
It also renders snapshot testing responsive elements useless, as the expected element will never be shallow rendered.

Reply:
An alternative to the requestAnimationFrame would be to call handleResize before the resizeObserver is attached with the current size of the of the container. Would this be acceptable?

```
  componentDidMount() {
    if (this.container) {
      this.handleResize(this.container.innerWidth);
      this.resizeObserver = new ResizeObserver((entries) => { this.handleResize(entries[0].contentRect.width); });
      this.resizeObserver.observe(this.container);
    } else {
      this.handleResize(window.innerWidth);
      window.addEventListener('resize', this.handleWindowResize);
    }
  }
```

Reply:
Calling handResize before observe might cause a triple render